### ISSUE TITLE
Unable to see full contents of modal shortcuts dialog

### ISSUE THREAD

**Issue by cpcallen** [rachel-fenichel/blockly-keyboard-experimentation#94]
Ideally the shortcuts dialog should be able to show all available shortcuts on one screenful, but failing this (e.g. on smaller screens) it should be possible to scroll the contents of the dialog using the keyboard only.

**Comment by cpcallen** [comment]
@bmxedd: I believe you've changed the layout to be more dynamic; I'm going to assign this to you for the moment in case there are other changes you wanted to make.  Then let's coordinate (possibly also with Ben) about the needed changes to make scrolling work via keyboard.

**Comment by kmcnaught** [comment]
Would it also be possible to have `Esc` close the help dialog? 

fwiw scrolling with arrow keys currently works _if you manage to get focus into the element_ but this doesn't happen by default

**Comment by bmxedd** [comment]
Both keyboard scrolling and Esc are possible when switching to the built in HTML dialog methods to handle open and closing the dialog. 

I have a pending CL.  @cpcallen am waiting on another pull request to be approved first before creating a new one.
